src/cli.rs

```diff
@@ -851,7 +856,35 @@ impl clap::Args for Exec {
                            fd -g 'test_*.py' -X vim\n\n  \
                        - Find all *.rs files and count the lines with \"wc -l ...\":\n\n      \
                            fd -e rs -X wc -l\
-                     "
+                    "
+                ),
+        )
+        .arg(
+            Arg::new("filter")
+                .action(ArgAction::Append)
+                .long("filter")
+                .short('f')
```

Comment:
I'm not sure if we should use a short option for this. After all we don't have a short option for `--exec` or `--exec-batch` which I think are probably more common than `--filter`.  And we may want to use `-f` for something else in the future (maybe a `--format` option? ). It is easier to add this alias later if desired than to remove it, so let's leave it off for now.